syscheck/check.go

```diff
@@ -17,11 +17,16 @@
  *
  */
 
-package sanity
+package syscheck
 
 var checks []func() error
 
-func Check() error {
+// CheckSystem ensures that the system is capable of running snapd and
+// snaps. It probe for e.g. that cgroup support is available and squashfs
+// files can be mounted.
+//
+// An error with details is returned if some check fails.
+func CheckSystem() error {
```

Comment:
syschec.CheckSystem, feels a bit awkward, maybe syscheck.Check or syscheck.SupportsSnapd() ?

Reply:
I find just Check even more awkward.  Our Supports* functions tend to return a bool. Instead of this I can only think of CheckSnapdIsSupported